airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/utils.py

```diff
@@ -135,3 +136,13 @@ def get_source_defined_primary_key(stream):
         catalog = json.loads(open(args.catalog).read())
         res = {s["stream"]["name"]: s["stream"].get("source_defined_primary_key") for s in catalog["streams"]}
         return res.get(stream)
+
+
+def serialize_to_date_string(date: str, date_format: str, date_type: str) -> str:
```

Comment:
Could you please give the examples of the dates for each condition?

Reply:
Added doc string